pennylane/devices/default_qubit.py

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This module contains the next generation successor to default qubit
+The default.qubit device is PennyLane's standard qubit-based device.
+
+This module contains the next generation successor to default qubit.
```

Comment:
```suggestion
```

default qubit is a successor to default qubit? Don't think we need this sentence anymore.